eZ/Bundle/EzPublishCoreBundle/Resources/config/routing/content.yml

```diff
@@ -0,0 +1,3 @@
+_ezplatform_download:
+    path: /content/download/{contentId}/{fieldIdentifier}/{downloadName}
```

Comment:
question here, this downloadName is mandatory? wouldn't be enought with contentId and fieldIdentifier? Then i would download the file with the name the file has...